Node/src/models/o365ConnectorCard.ts

```diff
@@ -0,0 +1,577 @@
+    private choicesData: teams.IO365ConnectorCardMultichoiceInputChoice[] = [];
+
+    constructor(protected session?: builder.Session) {
+        super(session);
+        (<teams.IO365ConnectorCardMultichoiceInput>this.data).choices = this.choicesData;
+    }
+
+    public isMultiSelect(flag: boolean): this {
+        (<teams.IO365ConnectorCardMultichoiceInput>this.data).isMultiSelect = !!flag;
+        return this;
+    }
+
+    public style(s: teams.IO365ConnectorCardMultichoiceInputStyle): this {
+        if (s) {
+            (<teams.IO365ConnectorCardMultichoiceInput>this.data).style = s;
+        }
+        return this;
+    }
+
+    public compactStyle(): this {
+        (<teams.IO365ConnectorCardMultichoiceInput>this.data).style = 'compact';
+        return this;
+    }
+
+    public expandedStyle(): this {
+        (<teams.IO365ConnectorCardMultichoiceInput>this.data).style = 'expanded';
+        return this;
+    }
+
+    public choices(list: teams.IO365ConnectorCardMultichoiceInputChoice[]|teams.IIsO365ConnectorCardMultichoiceInputChoice[]): this {
+        if (list) {
+            for (var i = 0; i < list.length; i++) {
+                var item = list[i];
+                if ((<teams.IIsO365ConnectorCardMultichoiceInputChoice>item).toChoice) {
+                    this.choicesData.push((<teams.IIsO365ConnectorCardMultichoiceInputChoice>item).toChoice());
+                 } else {
+                    this.choicesData.push(<teams.IO365ConnectorCardMultichoiceInputChoice>item);
+                 }    
+            }
+        }
+        return this;
+    }
+
+    protected get type(): string {
+        return 'multichoiceInput';
+    }
+}
+
+export class O365ConnectorCardMultichoiceInputChoice implements teams.IIsO365ConnectorCardMultichoiceInputChoice{
+    private data = <teams.IO365ConnectorCardMultichoiceInputChoice>{};
+
+    constructor(protected session?: builder.Session) {
+    }
+
+    public display(text: string|string[], ...args: any[]): this {
+        if (text) {
+            this.data.display = fmtText(this.session, text, args);
+        }
+        return this;
+    }
+
+    public value(text: string): this {
+        if (text) {
+            this.data.value = text;
+        }
+        return this;
+    }
+
+    public toChoice(): teams.IO365ConnectorCardMultichoiceInputChoice {
+        return this.data;
+    }
+}
+
+export function fmtText(session: builder.Session, prompts: string|string[], args?: any[]): string {
+    var fmt = builder.Message.randomPrompt(prompts);
+    if (session) {
+        // Run prompt through localizer
+        fmt = session.gettext(fmt);
+    }
+    return args && args.length > 0 ? sprintf.vsprintf(fmt, args) : fmt; 
+}
+
+export function o365ActionToPayload(obj: teams.IO365ConnectorCardActionBase): teams.IO365ConnectorCardActionBase{
+    if (obj.type) {
+        Object.defineProperty(obj, '@type', Object.getOwnPropertyDescriptor(obj, 'type'));
```

Comment:
I think it would. I'm doing my own placeholder implementation of this until the SDK is released and I'll try this. (Same with @id below.